projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/abilities/armor/snowdrift.json

```
@@ -0,0 +1,15 @@
+{
+  "modifier_id": "tconstruct:snowdrift",
+  "text": [
+    {
+      "text": "看看雪傀儡。它留下雪层的样子是不是很酷？你是不是也像和它一样留下雪层？"
+    }
+  ],
+  "effects": [
+    "在走过的路上留下雪",
+    "在温暖的生物群系不起作用",
+    "加持“延展”扩大覆雪范围",
```

```suggestion
    "通过“延展”升级来扩大覆雪范围",
```
@xingmot 提到的这个东西吧，表述的问题。